simiyu1/JavaDevsNairobi
# 155965911 create unit tests (In progress)

#### What does this PR do?
 - Create unit tests
#### Description of Task to be completed?
 - add dependencies for junit and mockito in build
 - create class MainActivityTests in res>tests folder
 
#### How should this be manually tested?
 - On navigation to the tests directory and you should see the [ MainActivityTests](https://github.com/simiyu1/JavaDevsNairobi/blob/ch-add-unit-tests/app/src/test/java/com/example/simiyunabende/javadevs/MainActivityTests.java) file

#### Any background context you want to provide?
 - There should be pre-commit hooks to check committed code. One of the areas checked should be whether the unit tests are passing.

#### What are the relevant pivotal tracker stories?
 - [#155965911](https://www.pivotaltracker.com/story/show/155965911)
